tensorflow/contrib/makefile/compile_nsync.sh

```diff
@@ -302,7 +308,9 @@ done
 case "$target_platform" in
 ios)    nsync_platform_dir="$nsync_builds_dir/lipo.$target_platform.c++11"
         mkdir "$nsync_platform_dir"
```

Comment:
remove previous line?

Reply:
I am not intended add the `-p` option to the command, let me remove it, thanks for reminding.